docs/resources/scanner.md

```diff
@@ -3,17 +3,17 @@ layout: "wallarm"
 page_title: "Wallarm: wallarm_scanner"
 subcategory: "Common"
 description: |-
-  Provides the resource to manage Wallarm Scanner scope.
+  Provides the resource to manage Wallarm scanner scope.
 ---
 
 # wallarm_scanner
 
-Provides the resource to manage the Wallarm Scanner scope. The scope defines company resources that must be scanned by the Wallarm dynamic application security testing (DAST) tool.
+Provides the resource to manage the Wallarm [scanner][] scope. The scope defines company resources that must be scanned by the Wallarm dynamic application security testing (DAST) tool.
```

Comment:
link?

Reply:
scope is incorrect term

Reply:
we do not mention DAST in official docs, what is it?

Reply:
Let's discuss.